src/components/Tabs/story.js

```diff
@@ -36,3 +36,106 @@ storiesOf('Next/Tabs', module)
       </Tabs>
     )
   })
+  .add('Compact', () => {
+    return (
+      <Box maxWidth={400}>
+        <Tabs p={3} compact>
+          <Tab title='Tab 1'>
+            <Txt mt={3}>Here is tab #1</Txt>
+          </Tab>
+          <Tab title='Tab 2'>
+            <Txt mt={3}>Here is tab #2</Txt>
+          </Tab>
+          <Tab title='Tab 3'>
+            <Txt mt={3}>Here is tab #3</Txt>
+          </Tab>
+          <Tab title='This is a very long tab name'>
+            <Txt mt={3}>Here is tab #1</Txt>
+          </Tab>
+          <Tab title='This is another very long tab name'>
+            <Txt mt={3}>Here is tab #2</Txt>
+          </Tab>
+          <Tab title='And this is a third very long tab name testing'>
+            <div>
+              <Txt mt={3}>
+                Hereeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee is tab #3
```

Comment:
Let's replace this with Lorem Ipsum as a more esthetic option :)

Reply:
agree :+1: